lib/cli/commands/listtrades.ts

```diff
@@ -0,0 +1,21 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { ListTradesRequest } from '../../proto/xudrpc_pb';
+
+export const command = 'listtrades [all]';
+
+export const describe = 'list all trades';
+
+export const builder = {
+  existing: {
```

Comment:
This should say "all" rather than "existing"

However I think it might actually be better to allow the caller to specify a number of results, rather than all or a fixed number. You can just make this a number called `limit` as you've done elsewhere, set a default value of 15 from the cli, and make it so a value of 0  (which is the value if it's unspecified through the grpc call) returns all trades.